libs/designsystem/src/lib/components/segmented-control/segmented-control.component.ts

```diff
@@ -150,4 +151,25 @@ export class SegmentedControlComponent<TItem extends SegmentItem = SegmentItem>
       });
     }
   }
+
+  focusNativeButton(event: UIEvent) {
+    (event.target as HTMLIonSegmentButtonElement)?.setFocus();
+  }
+
+  getTabIndex(item: TItem, index: number) {
+    // When focused prevent tab stop from inner native button to outer ion-segment-button:
+    if (this.ionSegmentElement.nativeElement.matches(':focus-within')) return -1;
+    // Allow tab stop on selected item:
+    if (item.id === this.value?.id) return null;
+    // Allow tab stop on first item if no value is set:
+    if (!this.value && index === 0) return null;
+    // Prevent tab stop on all other items:
+    return -1;
+  }
+
+  @HostListener('focusin', ['$event'])
+  @HostListener('focusout', ['$event'])
+  _noop() {
```

Comment:
Consider making this private so it does not show up in the component type
```suggestion
  private noop() {
```